### DISCUSSION TITLE
第三方插件

### DISCUSSION THREAD

**Discussion by bujinbufa** [MoeNetwork/Tieba-Cloud-Sign#264] [疑问]
插件库里的第三方插件，每日签到bark通知扩展，我把php都打包成zip上传提示     插件安装失败：插件包不合法，请确认此插件为百度贴吧云签到插件

**Comment by n0099** [comment]
这个？ https://github.com/Yuuuuu0/yuu_sign_bark
解压为目录`yuu_sign_bark`到plugins下
